LBL/LblWebClient/app/common/BaseRepository.ts

```diff
@@ -11,7 +11,7 @@
         constructor(http: angular.IHttpService, q: angular.IQService) {
             this.http = http;
             this.q = q;
-            this.baseUrl = "http://localhost:30285/api/";
+            //this.baseUrl = AppConstants.BaseApiUrl;
         }
 
         post(subUrl: string, data: any): angular.IPromise<any> {
```

Comment:
send  full URL  instead of sub. full url will be  constructed from our caller service class. 
**

> Service  layer  is the brain.

**